lib/origen_testers/igxl_based_tester/base.rb

```diff
@@ -717,7 +717,11 @@ def pattern_footer(options = {})
         $tester.align_to_last
         # cycle(:microcode => "#{$dut.end_of_pattern_label}:") if $dut.end_of_pattern_label
         if options[:end_in_ka]
+          if $tester.j750?
```

Comment:
Been a while editing base.rb and doing PRs in github, but I thought the whole point here was to not have have base handle all tester-specifics but have it handled in the child classes j750.rb and ultraflex.rb ?